server/controller/posts_controller.ts

```diff
@@ -19,4 +20,17 @@ export const getPosts = async (req : Request, res : Response, next : NextFunctio
     }catch(err){
         next(err);
     }
+}
+
+export const getPost = async (req : Request, res : Response, next : NextFunction) => {
+    try {
+        const post_id = parseInt(req.params.post_id);
+        if (isNaN(post_id)) {
+            throw ServerError.badRequest("Invalid post ID");
+        }
+        const post = await getPostInfo(post_id);
+        res.json({ post : post });
```

Comment:
*nit* 상태 코드를 명시해 주시면 좀 더 명확할 것 같습니다.

```suggestion
        res.status(200).json({ post : post });
```

Reply:
게시글 내용 보기는 빈값일 때 error 처리하는게 맞네요.
issue를 이전에 작성할 때 잘못 작성했고, issue만 보고 개발해서 이렇게 된 거 같아요...
지적 감사합니다. 이래서 PR이 참 좋은거 같아요 ㅠㅠ